Sources/Amplitude/Plugins/DestinationPlugin.swift

```diff
@@ -50,26 +49,13 @@ extension DestinationPlugin {
         return self.amplitude?.logger
     }
 
-    @discardableResult
-    func add(plugin: Plugin) -> Plugin {
-        plugin.amplitude = self.amplitude
-        timeline.add(plugin: plugin)
-        return plugin
-    }
-
-    func remove(plugin: Plugin) {
-        timeline.remove(plugin: plugin)
-    }
-
```

Comment:
@liuyang1520 sorry comment was in the wrong spot.

Reply:
No worries! I think this is ok to keep, as Kotlin SDK has [similar part](https://github.com/amplitude/Amplitude-Kotlin/blob/789feee5193659b1c92288b173c6f9395b639393/core/src/main/java/com/amplitude/core/platform/Plugin.kt#L61-L68). [`IdentityEventSender`](https://github.com/amplitude/Amplitude-Swift/blob/2088f7619f1c2180f3e66e9f760c3374cce0abb3/Sources/Amplitude/Plugins/AmplitudeDestinationPlugin.swift#L59) is using it for now, I saw this is removed in this PR. Not sure whether it has been used by any customer though, and I don't think this has been mentioned in the [docs](https://www.docs.developers.amplitude.com/data/sdks/ios-swift/#destination-type-plugin). @yuhao900914 , what do you think?

Reply:
This is needed if we are going to implement the experiment connector. We need to `add(plugin: IdentityEventSender())` at the end of the destination plugin setup. 
But currently `IdentityEventSender()` is doing nothing. It's just a placeholder. If we remove that logic, we can safely remove this also.